spec/models/skin_spec.rb

```diff
@@ -164,13 +164,42 @@
       expect(@skin.errors[:base].join(' ').match(/upload a screencap/)).to be_truthy
     end
 
-    it "should only allow valid media types" do
-      @skin.media = ["foobar"]
-      expect(@skin.save).not_to be_truthy
-      expect(@skin.errors[:base]).not_to be_empty
-      @skin.media = %w(screen print)
-      expect(@skin.save).to be_truthy
-      expect(@skin.errors[:base]).to be_empty
+    context "when a media query is provided" do
+      {
+        "allows all" => "all",
+        "allows screen" => "screen",
+        "allows handheld" => "handheld",
+        "allows speech" => "speech",
+        "allows print" => "print",
+        "allows braille" => "braille",
+        "allows embossed" => "embossed",
+        "allows projection" => "projection",
+        "allows tty" => "tty",
+        "allows tv" => "tv",
+        "allows only screen and (max-width: 42em)" => "only screen and (max-width: 42em)",
+        "allows only screen and (max-width: 62em)" => "only screen and (max-width: 62em)",
+        "allows (prefers-color-scheme: dark)" => "(prefers-color-scheme: dark)",
+        "allows (prefers-color-scheme: light)" => "(prefers-color-scheme: light)"
+      }.each_pair do |description, media_query|
```

Comment:
I like that you're listing them explicitly instead of using `Skin::MEDIA`, so we don't accidentally break both code and test at once.

You can do this for less duplication:
````rb
      [
        "all",
        "(prefers-color-scheme: light)"
      ].each do |media_query|
        it "allows #{media_query}" do
````

Reply:
🤦‍♂ 

As I was writing this, I was like, "I know I'm doing something inefficiently here, but I don't know what. Hopefully someone can tell me."